Cython/Compiler/Parsing.py

```diff
@@ -2340,7 +2342,25 @@ def p_statement_list(s, ctx, first_statement = 0):
     pos = s.position()
     stats = []
     while s.sy not in ('DEDENT', 'EOF'):
-        stat = p_statement(s, ctx, first_statement = first_statement)
+        # When processing statements in a fault-tolerant mode, it's possible
```

Comment:
Please move this comment right after the `if s.fault_tolerant`.

Reply:
with scope identation?